metric-anomaly-detector/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
     implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.28")
     implementation("com.typesafe:config:1.4.1")
     implementation("org.hypertrace.gateway.service:gateway-service-api:0.1.59")
+    implementation("org.hypertrace.gateway.service:gateway-service-baseline-lib:0.1.167")
```

Comment:
Looks like we accidentally have jumped the version from 0.1.65 to 0.1.166 onwards - https://github.com/hypertrace/gateway-service/releases/

Reply:
Pl. next time move to `0.2.0` or something.